app/assets/stylesheets/syntax.scss

```diff
@@ -31,6 +40,30 @@ div.highlight {
   @media (min-width: $breakpoint-m) {
     padding: var(--su-6);
   }
+
+  button {
+    right: 4px;
+    top: 4px;
+    height: 19px;
+    display: none;
+    position: absolute;
+  }
+  &:hover {
+    button {
+      display: block;
+    }
+  }
+}
+
+// *Definitely* doesn't belong here, just tossed it here and probably should be the fully-reusable modal bg.
+.crayons-fullscreenbg {
```

Comment:
I know this is closed but I thought I'd share a little trick.
Rather than spawning in a whole new div to the dom to could do something like

```scss
&.crayons-fullscreen {
  position: fixed;
  top: calc(50px + 3vh);
  left: calc(5px + 3vw);
  bottom: 3vh;
  right: calc(5px + 3vh);
  z-index: 150;

  &--backdrop {
    box-shadow: 0 0 0 200vmax rgba(0,0,0,0.8);
  }
}
```

Now you can have  a div like this.

```html
<div class="crayons-fullscreen crayons-fullscreen--backdrop" >CONTENT</div>
```

![result](https://user-images.githubusercontent.com/3534427/93357148-e78eb300-f837-11ea-9e44-524564b66d3a.png)